tests/rptest/services/redpanda.py

```diff
@@ -361,7 +361,7 @@ def __init__(self):
 
 class RedpandaService(Service):
     PERSISTENT_ROOT = "/var/lib/redpanda"
-    DATA_DIR = os.path.join(PERSISTENT_ROOT, "data")
+    HOST_PERSISTENT_ROOT = "/host_var"
```

Comment:
couple of things:
  * I think `PERSISTENT_ROOT` should be the one that gets placed on other paths, not just `data/` inside it.
  * Exposing a variable instead of hard-coding it
  * Having `/var/lib/redpanda` by default if the variable is empty

Reply:
the `/host_var` is not highly used, it is defined here and in the docker-compose configuration. is it required exposing a variable for it?

Reply:
Since I changed the whole logic leaving intact the RedpandaService I'll resolve this conversation